ansible_collections/arista/avd/plugins/plugin_utils/eos_validate_state_utils/csv_report.py

```diff
@@ -41,12 +41,12 @@ def generate_rows(self) -> Generator[dict, None, None]:
             yield {
                 **result,
                 "messages": ";".join(result["messages"]),
-                "test_categories": ";".join(result["test_categories"]),
+                "categories": ";".join(result["categories"]),
             }
 
     def generate_report(self) -> None:
         """Generate and writes the CSV report using the collected test results."""
-        fieldnames = ["test_id", "node", "test_categories", "test_description", "test", "result", "messages"]
+        fieldnames = ["id", "dut", "categories", "test", "description", "inputs", "result", "messages"]
```

Comment:
Should we match the field names with the markdown report? i.e dut -> device under test? Or in the CSV we keep things brief?
Also, this might be considered a breaking change since this is structured data that might be used to populate other systems.
@gmuloc @ClausHolbechArista

Reply:
Since anta is still in preview, any changes to the csv is fine.
IMO headings should be short and easy to parse in code. So `dut` is better than `device under test`

Reply:
Let's track for 5.0 to include in the release notes that the heading for CSV are now changing.

Reply:
@ClausHolbechArista - Since we are keeping both methods (current ansible logic and ANTA) in 4.x then yes I agree we can make the changes now.